server/controllers/personal/personalController.ts

```diff
@@ -93,10 +92,26 @@ export default class PersonalController {
           const { editField } = req.body
           const prisonPerson = await this.personalPageService.getPrisonPerson(clientToken, prisonerNumber, true)
 
-          const height = parseInt(editField, 10)
-          if (Number.isNaN(height) || height <= 0) {
+          const height = editField ? parseInt(editField, 10) : 0
+
+          const validations = [
+            {
+              validation: () => editField === '' || editField === undefined || !Number.isNaN(height),
```

Comment:
As you've already checked this above, perhaps you could just do `editField === 0 || !Number.isNaN(height)`